packages/core/admin/admin/src/content-manager/components/DynamicTable/PublicationState/PublicationState.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+import styled from 'styled-components';
+import { Status, Typography } from '@strapi/design-system';
+
+import { getTrad } from '../../../utils';
+
+const StyledStatus = styled(Status)`
+  width: min-content;
```

Comment:
I think you can pass `width` directly to `Status` as it allows CSS properties

Reply:
Done in 27753fc03bd24d35a8fc6f3fcd24a43597e5f1c9